README.md

```diff
@@ -45,6 +45,18 @@ You want to specify the following options:
 
 Finally, click "Add webhook".
 
+### The GitHub App
+
+To manage GitHub instances, Sheriff requires you to create a GitHub App that gets installed in the desired Org.
```

Comment:
We should also probably document what permissions the app needs.  Specifically:

```md
Org:
administration:write
contents:read
metadata:read

Repo:
members:write
```

Reply:
ezpz